notebooks/237-typo-detector/README.md

```diff
@@ -0,0 +1,58 @@
+# Convert and Optimize Typo Detector with OpenVINO™
+
+Typo detection in AI is a process of identifying and correcting typographical errors in text data using machine learning algorithms. The goal of typo detection is to improve the accuracy, readability, and usability of text by identifying and correcting mistakes made during the writing process.
+
+A typo detector takes a sentence as an input and identify all typographical errors such as misspellings and homophone errors.
+
+This tutorial provides how to use the [Typo Detector](https://huggingface.co/m3hrdadfi/typo-detector-distilbert-en) from the [Hugging Face Transformers](https://huggingface.co/docs/transformers/index) library to perform the above task.
+
+The model has been pretrained on the [NeuSpell](https://github.com/neuspell/neuspell) dataset.
+
+<img src=https://user-images.githubusercontent.com/80534358/224564463-ee686386-f846-4b2b-91af-7163586014b7.png>
+
+</br>
+
+The notebook provides two methods to use the typo detector in the OpenVino runtime. I've demonstrated both methods so that you can experience both the loading of compiled models and loading models in other frameworks and converting them.
+
+1. Use the [Hugging Face Optimum](https://huggingface.co/docs/optimum/index) library to load the compiled model in OpenVino IR format. Then create a pipeline with the loaded model to run inference.
+
+2. Load the model and convert to ONNX and then to OpenVino IR.
+   First the Pytorch model is convereted to the ONNX format and then the [Model Optimizer](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) tool is used to convert to [Openvino IR format](https://docs.openvino.ai/latest/openvino_ir.html). This method provides much more insight to the openvino environment and applications.
```

Comment:
```suggestion
   First the Pytorch model is convereted to the ONNX format and then the [Model Optimizer](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) tool is used to convert to [OpenVINO IR format](https://docs.openvino.ai/latest/openvino_ir.html). This method provides much more insight to how to set up a pipeline from model loading to model converting, compiling and running inference with OpenVINO, so that you could conveniently use OpenVINO to optimize and accelerate inference for other deep-learning models.
```